### DISCUSSION TITLE
Build fails when there are three or more Assembler CICS modules to compile

### DISCUSSION THREAD

**Discussion by jminthorne-axc** [IBM/dbb-zappbuild#554] [Q&A]
When I have a list of assembler modules with items where isCics=true, the build fails on the third item. The build ends, and no error message is provided. If I run the build for the "failed" module, either without other items or the item is not the third in the list, the module builds without issue.

**Comment by mhardin18** [comment]
You need to provide more information.  Can you provide the log output, with verbosity, of the failing builds?
